vscode-dotnet-runtime-extension/package.json

```diff
@@ -11,7 +11,7 @@
 	"author": "Microsoft Corporation",
 	"displayName": ".NET Install Tool",
 	"description": "This extension installs and manages different versions of the .NET SDK and Runtime.",
-	"appInsightsKey": "02dc18e0-7494-43b2-b2a3-18ada5fcb522",
+	"connectionString": "InstrumentationKey=02dc18e0-7494-43b2-b2a3-18ada5fcb522;IngestionEndpoint=https://westus2-0.in.applicationinsights.azure.com/;LiveEndpoint=https://westus2.livediagnostics.monitor.azure.com/;ApplicationId=e8e56970-a18a-4101-b7d1-1c5dd7c29eeb",

```

Comment:
One thing that is slightly concerning is that all telemetry calls will be going to the `westus` Azure region despite our users being global. Can you check with them if there is a set of urls we should be using based on user locale instead?

Reply:
That's a good question. The devdiv data team shared that this is intentional as there is only one endpoint. I discussed this further and apparently DotnetCLI and AzureCLI use 1 endpoint in western US as well and they state they havent had any scalability issues so far with this methodology. I did emphasize the regional issues we've solved in the past as being a point of concern.